presto-hive/src/test/java/com/facebook/presto/hive/metastore/glue/TestHiveClientGlueMetastore.java

```diff
@@ -49,4 +49,28 @@ public void testRenameTable()
     {
         // rename table is not yet supported by Glue
     }
+
+    @Override
+    public void testUpdateTableColumnStatistics()
+    {
+        // column statistics are not supported by Glue
```

Comment:
can you throw SkipException here and elsewhere?

Reply:
it isn't our convention to do so

Reply:
It is not, but I think it should be. Telling people that test is passing while it is not possible for test to pass might be a bit misleading. I hope raising SkipException might have a better developer experience.

Reply:
This is provided that someone reads the list of passing tests. I don't.
I do read lists of failing tests only. And Jenkins's testng plugin lists all the skipped tests, which isn't entirely interesting when the test is skipped because the functionality simply does not exist

Reply:
Usually we throw `SkipException` if there is no easy way of disabling a test in any other way. (throwing it from somewhere deep inside the mock objects)